### PR TITLE
Isarr class - Isarr following Graey and shooting

### PR DESCRIPTION
**Pull request by markfilliph**
- Isarr has a distance to chase Graey
- Isarr has also a distance to flee and shoot

### CODE REVIEW

**Review by Seanlegroulx — APPROVED** [review]
golden!

**Review by ValeriaRincon — APPROVED** [review]
Approved

**Review by TroyDexter — APPROVED** [review]
Approved

File: Assets/Scripts/FinalBoss/Enemy_Isarr.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = System.Random;
+
+/* script created following this tutorial: https://www.youtube.com/watch?v=_Z1t7MNk0c4
```
Comment (DoopweeSLC) [inline Assets/Scripts/FinalBoss/Enemy_Isarr.cs:7]: great job documenting the tutorial source!

**Pull request merged**